src/assets/styles/app.css

```diff
@@ -959,6 +963,27 @@ aside h2 {
     margin-block-start: 5rem;
 }
 
+@media all and (max-width: 24rem) {
+    .site-brand {
+        margin-block: clamp(0.5rem, (2 - var(--fl-textSize-factor, 1)) * 1.28125rem, 1.28125rem);
+    }
+
+    .site-header .wrapper {
+        padding-left: 10px;
+        padding-right: 10px;
```

Comment:
Shall `rem` be used instead of `px`?

Reply:
I used `px` since otherwise, the padding would increase with the text size, which is the opposite of what we're trying to achieve with this rule override.

According to @jobara while we were working on Phase 1 of the redesign (from the _Fluid Design_ channel on Dec. 14, 2021):

> the padding-inline was specifically set to a fixed size. When it was set to rem values it would grow with font size changes in UIO and take up too much space, specifically on mobile.

Reply:
Makes sense. Thanks, @BlueSlug.